im-gespraech-buber-gesellschaft.csl

```diff
@@ -0,0 +1,651 @@
+        <label strip-periods="true" prefix=", " suffix=". " variable="page" form="short"/>
+        <text variable="page"/>
+        <text variable="note" prefix=" [erstmals " suffix="]"/>
+      </if>
+      <else-if type="entry-encyclopedia" match="any">
+        <label variable="locator" form="short" suffix=". " strip-periods="true"/>
+        <text variable="page"/>
+        <text variable="note" prefix=" [erstmals " suffix="]"/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="citation-locator">
+    <choose>
+      <if type="manuscript" match="none">
+        <label variable="locator" form="short" suffix=". " strip-periods="true"/>
+      </if>
+    </choose>
+    <text variable="locator"/>
+  </macro>
+  <macro name="date">
+    <choose>
+      <if match="any" variable="issued">
+        <date form="text" variable="issued"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="collection-editor">
+    <names variable="collection-editor" prefix="Hrsg. von ">
+      <name delimiter="/" delimiter-precedes-last="always"/>
+    </names>
+  </macro>
+  <macro name="volume">
+    <group suffix=".">
+      <text term="volume" form="short" suffix=" "/>
+      <text variable="volume"/>
+    </group>
+  </macro>
+  <macro name="Band aus Reihe">
+    <choose>
+      <if match="any" variable="title">
+        <group>
+          <text macro="editor" prefix=" "/>
+          <group prefix=" (" suffix=") ">
+            <group>
+              <text variable="collection-title"/>
+              <choose>
+                <if match="any" variable="collection-editor">
+                  <text value=". "/>
+                </if>
+              </choose>
+            </group>
+            <text macro="collection-editor"/>
+            <text variable="volume" prefix=", Bd. "/>
+          </group>
+        </group>
+      </if>
+      <if match="none" variable="title">
+        <group>
+          <group>
+            <text variable="collection-title" font-style="italic"/>
+            <choose>
+              <if match="any" variable="collection-editor">
+                <text value=" ."/>
+              </if>
+            </choose>
+          </group>
+          <text macro="collection-editor"/>
+          <text macro="volume" prefix=", "/>
+          <text macro="editor"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="container-title">
+    <choose>
+      <if match="any" variable="container-title">
+        <text macro="editor" prefix=" "/>
+        <group prefix=" (" suffix=") ">
+          <group>
+            <text variable="collection-title"/>
+            <choose>
+              <if match="any" variable="collection-editor">
+                <text value=". "/>
+              </if>
+            </choose>
+          </group>
+          <text macro="collection-editor"/>
+          <text variable="volume" prefix=", Bd. "/>
+        </group>
+      </if>
+      <if match="none" variable="container-title">
```

Comment:
again--can't have two consecutive ifs. Not sure what you're trying to do here, though.